.github/workflows/gradle.yml

```
@@ -0,0 +1,27 @@
+
+      - name: 2. JDK 17 세팅
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+
+      - name: 3. gradlew 실행 권한 부여
+        run: chmod +x ./gradlew
+
+      - name: 4. Gradle 빌드 실행 (테스트 제외)
+        run: ./gradlew clean build -x test
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)